pkg/verify/verify.go

```diff
@@ -74,7 +72,7 @@ var Reject Interface = rejectVerifier{}
 // if each provided public key has signed the release image digest. The signature may be in any
 // store and the lookup order is internally defined.
 //
-func LoadFromPayload(update *payload.Update) (Interface, error) {
+func LoadFromPayload(update *payload.Update, clientBuilder clientBuilder) (Interface, error) {
```

Comment:
I don't think an argument of type `package private` is a good pattern for `public` function

Reply:
Updated to export ClientBuilder interface type/